### DISCUSSION TITLE
Website Not Displaying Correctly on GitHub Pages

### DISCUSSION THREAD

**Discussion by yararajjoub** [community/community#84077] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello!

I am facing an issue with my personal website hosted on GitHub Pages. When I visit my GitHub Pages link (https://yararajjoub.github.io/My_Portfolio/), I only see the README content instead of the actual website content.

The main HTML file is named index.html, and it is located in the root directory of my repository. I have selected Main/root from Pages.

Could you help me with that?

**Comment by seyLu** [comment, marked as answer]
#### Hi @yararajjoub,

I've taken a look at your commit history and noticed that you created a `static.yaml` workflow and then reverted that change afterwards. First, on your repo settings, change build and deploy source from workflow to branch, then select `/(root)`.

GitHub pages looks for `index.html` at the root directory, and if it can't find it, it will look for `index.md` or in your case `README.md`.

Root directory, meaning, having a folder structure like:

```bash
images
 |- ...
index.html
style.css
README.md
```

and not:
![image](https://github.com/community/community/assets/98249191/4438f2ce-f9f2-4f4f-98ce-e69d1d94fd78)

You can read more at [GitHub Pages docs: Creating your site](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site#creating-your-site).

**Reply by yararajjoub** [reply]
Thank you very much, that was helpful!
